### DISCUSSION TITLE
Share your installations - show and tell

### DISCUSSION THREAD

**Discussion by rdz-oss** [rdz-oss/BattyBirdNET-Pi#43] [General]
Got a new setup of BattyBirdNET-Pi or similar? Want to share tips? This is the place!

**Comment by rdz-oss** [comment]
**Lesser Horseshoe bat live from one of their last roosts in Germany!**

I have been partnering with people from The LBV in Bavaria to setup a BattyBirdNET-Pi near a roost of Lesser Horseshoe bats that they take care of. A very cool experience with very cool people protecting very cool bats.

The live-stream is not yet public, but there is already more than 30GB audio data and bat activity data. Includes mother/pup social calls. Running at 384kHz 24/7 on a Pi 5 and usb audiomoth.

<img width="965" alt="Screenshot 2024-08-03 at 15 18 42" src="https://github.com/user-attachments/assets/50b348f9-82b0-4635-84e4-0f2d8fc0ea4a">

<img width="790" alt="Screenshot 2024-08-03 at 15 19 57" src="https://github.com/user-attachments/assets/c44d126d-eb21-44df-8269-2a96f76f4b91">

Want to see them live? Here is their camera stream:

https://www.lbv.de/ratgeber/naturwissen/tier-webcams/kleine-hufeisennase-banzerhaus/

And the live audio analysis? Running. Live sound via the cam feed - in progress.

**Comment by Inachis** [comment]
A lot of 'day-flying' bats? The spectrogram shows a regular pulse maybe triggering false detection?

I was about to ask a separate question but your image illustrates the same problem - why does the 'pi chart' have a gap between 22:30 and 23:30? I see exactly the same...looking in the database there are many detections between 22:30 and 23:30.

Kelvin

**Reply by rdz-oss** [reply]
Hi @Inachis ,
the mic is placed in a rooftop that is a roost. On average, that is their call activity. Also during the day. They hang about and also chirp when cleaning themselves etc.
The line does not trigger the classifier - it is in the noise set and ignored.
The gap is an artifact that comes from the binning algorithm. Can be fixed.

BR
Richard

**Reply by Inachis** [reply]
Hi Richard,
Thanks for clarifying the daytime activity.

Cheers,
Kelvin

**Reply by rdz-oss** [reply]
Kelvin, do I remember correctly about the canal? Are you still interested in recording there? The system has come a long way.

BR
Richard

**Reply by Inachis** [reply]
Yes, Richard, canal monitoring still in plan but getting to a mobile setup is taking longer than I'd hoped; also have yet to source another suitable ultrasonic microphone.
